client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts

```diff
@@ -67,6 +70,11 @@ const useInteractionsTab = ({ interactions, setInteractions }: useInteractionsTa
         axios.get(`/intersections/contactEvent/${epidemiologyNumber}`)
             .then((result) => {
                 const allInteractions: InteractionEventDialogData[] = result.data.map(convertDBInteractionToInteraction);
+                setAreThereContacts(
```

Comment:
I think placing the result of the reduce on a variable will improve readability here